examples/super_resolution/model/README.md

```diff
@@ -0,0 +1,15 @@
+Download 
+[srgan_96_4](https://drive.google.com/file/d/1WLbsFWhlSWF2FPFSwu5e7WkpXvzv2TYc/view?usp=sharing),
+[srgan_128_4](https://drive.google.com/file/d/1aZwhsLeXIEW4P62ncoDZ9dpsdaqG8Oym/view?usp=sharing),
+and put them here.
+
+The model files are:
+
+```txt
+srgan_96_4.tflite
+srgan_128_4.tflite
+```
+
+These models are trained by [TensorLayer](https://github.com/tensorlayer/tensorlayer) and then converted to TfLite format.
+
+Check out [GreyZzzzzzXh/srgan/convert_to_tflite](https://github.com/GreyZzzzzzXh/srgan/tree/master/convert_to_tflite) for details about training and convertion.
```

Comment:
@GreyZzzzzzXh , could you please put the training and conversion steps into the README.md here? It would be good to have one place to host the necessary materials.

Reply:
Sounds good, I will do that.